docs/Introduction/tutorials/intermediates-tutorial.md

```diff
@@ -99,7 +100,13 @@ bytes32 private s_keyHash;
 uint256 private s_fee;
 ```
 
-For the contract to keep track of addresses that roll the dice, the contract will need to use mappings. [Mappings](https://medium.com/upstate-interactive/mappings-in-solidity-explained-in-under-two-minutes-ecba88aff96e) are unique `key => value` pair data structures similar to hash tables in Java.
+Bellow are the values for Kovan testnet network. They will be updated in the next step, the `constructor`.
```

Comment:
```suggestion
Below are the values for Kovan testnet network. They will be updated in the `constructor`.
```

Reply:
Also, would it make sense to move information about values for Kovan testnet network to the next section? It may meld better since we introduce the constructor in the next section as well